EMS/core-bundle/src/Resources/views/user/index.html.twig

```diff
@@ -16,4 +16,4 @@
     }) }}
 
     {{ parent() }}
-{% endblock %}
\ No newline at end of file
+{% endblock %}
```

Comment:
When I delete this space, it adds again automatically. Need to run phpcs, but I cant run phpcs in a version higher than 8.1 ( using 8.2 now )